src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/AddPackageCommandUtility.cs

```diff
@@ -113,6 +114,32 @@ public static List<PackageSource> EvaluateSources(IList<PackageSource> requested
                     }
                 }
 
+                HashSet<string> currentPackagesSources = packageSources.Select(p => p.Source).ToHashSet(StringComparer.OrdinalIgnoreCase);
+
+                if (additionalSources?.Any() ?? false)
```

Comment:
The following is easier to read: 

```cs
                if (additionalSources?.Any() == true)
```

It's also what's used heavily in `AddPackageReferenceCommandRunner`.

Reply:
Ok. Addressed.